src/renderer/i18n/en/index.ts

```
@@ -12,7 +12,7 @@ import netstatus from './netstatus'
 import pools from './pools'
 import poolShares from './poolshares'
 import routes from './routes'
-import runePool from './runePool'
+import protocolPool from './protocolPool'
```

ESLint

[error] `./protocolPool` import should occur before import of `./routes`

(import/order)